### DISCUSSION TITLE
Other language doesn't generate.

### DISCUSSION THREAD

**Discussion by MaiChaMH** [lukaszliniewicz/Pandrator#14] [Q&A]
I'm using the 0.0.7 version with xtts (installed through one click exe), and English seems to be doing fine. But when I select another language (zh-cn & ja specifically) it doesn't generate. There's nothing happening after pressing the Start generation button, and when I click the cancel generation button after the start button it says that no process is happening (also went task manager to double check, no cpu process).

**Comment by lukaszliniewicz** [comment]
I see you closed the issue, have you found a solution? I tested Polish and German and it worked.
